src/styles/index.scss

```diff
@@ -1,3 +1,45 @@
-body {
+@import 'variables';
+@import 'media';
+@import 'stars';
+@import 'header';
+@import 'nav';
+@import 'card';
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
```

Comment:
Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser
to read your web document